controller/pkg/extensions/utils.go

```diff
@@ -0,0 +1,51 @@
+package extensions
+
+import (
+	"fmt"
+	"strings"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/kong/gateway-operator/pkg/consts"
+
+	operatorv1beta1 "github.com/kong/kubernetes-configuration/api/gateway-operator/v1beta1"
+	konnectv1alpha1 "github.com/kong/kubernetes-configuration/api/konnect/v1alpha1"
+)
+
+// ValidateExtensions validates the extensions referenced by the provided DataPlane and returns
+// a condition indicating if the extensions are supported or not.
+func ValidateExtensions(dataplane *operatorv1beta1.DataPlane) *metav1.Condition {
```

Comment:
Can we add a UT for this `func`?

Reply:
here it is: 64fd31ca3644b928d814fd0100d34222c325fb8f